ner_v2/detectors/temporal/time/en/time_detection.py

```diff
@@ -1,7 +1,16 @@
 import re
 import datetime
-from ner_v2.detectors.temporal.constant import AM_MERIDIEM, PM_MERIDIEM, TWELVE_HOUR, EVERY_TIME_TYPE
-from ner_v2.detectors.temporal.utils import get_timezone
+import collections
+import pandas as pd
+import os
+from ner_v2.detectors.temporal.constant import AM_MERIDIEM, PM_MERIDIEM, TWELVE_HOUR, EVERY_TIME_TYPE, \
+    TIMEZONE_VARIANTS_CONSTANT_FILE, TIMEZONES_CONSTANT_FILE, TIMEZONE_VARIANTS_VARIANTS_COLUMN_NAME, \
+    TIMEZONE_VARIANTS_VALUE_COLUMN_NAME, TIMEZONES_CODE_COLUMN_NAME, TIMEZONES_ALL_REGIONS_COLUMN_NAME, \
+    TIMEZONES_PREFERRED_REGION_COLUMN_NAME
+from ner_v2.detectors.temporal.utils import get_timezone, get_list_from_pipe_sep_string
+from ner_v2.constant import LANGUAGE_DATA_DIRECTORY
+
+TimezoneVariants = collections.namedtuple('TimezoneVariant', ['value'])
```

Comment:
Why are you using NamedTuple only for one param? 

if you want to you can use it to store the preferred value also.

Reply:
If we want to store everything, we can add **all regions** too. But, it will load everything for all TimezoneVariants into **timezones_map**. We can use dict instead of NamedTuple here.

Reply:
okay